src/responsive-child/responsive-child.js

```diff
@@ -1,40 +1,63 @@
-require("document-register-element");
+require("core-js/es/reflect");
+require('@webcomponents/custom-elements');
 require("./responsive-child.less");
 var makeEvent = require("../makeEvent");
 var Guest = require("./guest.js");
 
 var proto = Object.create(HTMLElement.prototype);
 var bodyProto = Object.create(HTMLBodyElement.prototype);
-bodyProto.createdCallback = proto.createdCallback = function() {
-  var self = this;
-  var guest = this.guest = new Guest(this, function(data) {
-    //message from the host
-    var event = makeEvent("message", { data: data });
-    self.dispatchEvent(event);
-  });
-  //set a default interval, but interval=0 still disables it.
-  var interval = this.getAttribute("interval") || 100;
-  interval *= 1;
-  if (interval) {
-    var height = null;
-    var loop = function() {
-      var h = self.offsetHeight;
-      // don't message if nothing has changed
-      if (h != height) guest.notify("interval");
-      height = h;
-      setTimeout(loop, interval);
-    };
-    loop();
-  }
+
+
+function ResponsiveChild() {
```

Comment:
Is there a reason not to just switch to the class-based syntax if you're going to V1?